internal/action/version.go

```diff
@@ -76,30 +77,43 @@ func versionInfo(ctx context.Context, v versioner) string {
 func (s *Action) checkVersion(ctx context.Context, u chan string) {
 	if disabled := os.Getenv("CHECKPOINT_DISABLE"); disabled != "" {
 		u <- ""
+		debug.Log("remote version check disabled by CHECKPOINT_DISABLE")
 		return
 	}
 
-	if strings.HasSuffix(s.version.String(), "+HEAD") || protect.ProtectEnabled {
-		// chan not check version against HEAD or
+	force := os.Getenv("GOPASS_FORCE_CHECK") == "true"
```

Comment:
`CHECKPOINT_DISABLE` can be any non-null string, but `GOPASS_FORCE_CHECK` must be `"true"`, maybe we should unify?

Reply:
Good idea.